lib/order.rb

```diff
@@ -0,0 +1,51 @@
+class Order
+  
+  ERR_INVALID_ORDER = "The selected dish is not on the menu".freeze
+  ERR_EMPTY_ORDER = "The order is empty".freeze
+  
+  attr_reader :dishes, :total
+  
+  def initialize(menu)
+    @dishes = {}
+    @total = 0
+    @menu = menu
+  end
+  
+  def add(dish, qty = 1)
+    fail ERR_INVALID_ORDER unless menu.contains? dish
+    dishes[dish] = qty
```

Comment:
what happens if you want to add more than 1 quantity of the same dish?

```ruby
order.add(:pizza, 2)
order.add(:pizza, 1)
```
I would expect to receive 3 pizzas.